public/app/features/dashboard-scene/saving/provisioned/PreviewBannerViewPR.test.tsx

```diff
@@ -18,6 +18,12 @@ jest.mock('app/features/provisioning/hooks/usePullRequestParam', () => ({
   usePullRequestParam: jest.fn(),
 }));
 
+jest.mock('app/features/provisioning/utils/selectors', () => ({
```

Comment:
I _think_ this is probably needed because of a circular dependency in the code, or similar to https://github.com/reduxjs/reselect/issues/169

We can get the tests passing without this mock if we refactored the selectors to be more like

```ts
const getBaseSelector = () => provisioningAPI.endpoints.listRepository.select({});

export const selectAllRepos = () => createSelector(getBaseSelector(), (result) => result.data?.items || emptyRepos);
```

etc.

If I do that and follow through with necessary refactors elsewhere, then the tests will pass